src/main/java/org/sonarlint/intellij/core/ProjectBindingManager.java

```diff
@@ -47,6 +56,36 @@ public ProjectBindingManager(Project project, SonarLintEngineManager engineManag
     this.console = console;
   }
 
+  @Override
+  public void initComponent() {
+    JobScheduler.getScheduler().scheduleWithFixedDelay(new Runnable() {
+      @Override
+      public void run() {
+        ProjectBindingManager.this.hasGlobalUpdates = false;
+        ProjectBindingManager.this.hasUpdates = false;
+        ProjectBindingManager.this.changelog.clear();
+        if (projectSettings.isBindingEnabled()) {
+          ConnectedSonarLintEngine engine = getConnectedEngine();
+          ServerConfiguration serverConfiguration = SonarLintUtils.getServerConfiguration(getSonarQubeServer());
+          StorageUpdateCheckResult checkForUpdateResult = engine.checkIfGlobalStorageNeedUpdate(serverConfiguration, null);
+          if (checkForUpdateResult.needUpdate()) {
+            ProjectBindingManager.this.hasGlobalUpdates = true;
+            ProjectBindingManager.this.hasUpdates = true;
+            ProjectBindingManager.this.changelog.addAll(checkForUpdateResult.changelog());
+          }
+          StorageUpdateCheckResult moduleUpdateCheckResult = engine.checkIfModuleStorageNeedUpdate(serverConfiguration, projectSettings.getProjectKey(), null);
+          if (moduleUpdateCheckResult.needUpdate()) {
+            ProjectBindingManager.this.hasUpdates = true;
+            ProjectBindingManager.this.changelog.addAll(moduleUpdateCheckResult.changelog());
+          }
+          if (ProjectBindingManager.this.hasUpdates) {
+            notifications.notifyServerHasUpdates(projectSettings.getServerId(), ProjectBindingManager.this.changelog, engine, getSonarQubeServer(), !hasGlobalUpdates);
+          }
+        }
+      }
+    }, 10, 24 * 60 * 60, TimeUnit.SECONDS);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Cast one of the operands of this multiplication operation to a "long". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS2184)